driver/registry_default.go

```diff
@@ -344,21 +344,11 @@ func (m *RegistryDefault) selfServiceStrategies() []any {
 }
 
 func (m *RegistryDefault) strategyRegistrationEnabled(ctx context.Context, id string) bool {
-	switch id {
-	case identity.CredentialsTypeCodeAuth.String():
-		return m.Config().SelfServiceCodeStrategy(ctx).PasswordlessEnabled
-	default:
-		return m.Config().SelfServiceStrategy(ctx, id).Enabled
-	}
+	return m.Config().SelfServiceStrategy(ctx, id).Enabled
```

Comment:
Hm, that's true... 

I think the decision to deviate from the standard "one `enabled` property to enable a strategy" was a mistake, hence this change. 

WDYT about this migration strategy for the Ory Network: If a project has `code.passwordless_enabled: true` set to true, but `code.enabled: false`, we set `code.enabled: true`. 

I doubt this would affect many projects, since `code.enabled` is `true` by default.

Reply:
sounds good. let's sync on that quickly.

Reply:
As discussed personally, I added a switch to the configuration layer, to force-enable the code method, if `passwordless_enabled` is true: https://github.com/ory/kratos/pull/3682/commits/523811886eef62cbf699d06c85eb83be2d5d14db